### PR TITLE
[v1.x] [CD] Add Aarch64 to cd

### PR DESCRIPTION
**Pull request by mseth10**
## Description ##
This PR adds AArch64 support to MXNet CD pipeline.

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
 Following major changes are made as part of this PR:
- Added u18 aarch64 nodes to Jenkins CI
- Added restricted g4 node to be used for quantization tests in CD pipeline
- Removed p3 nodes not being used anywhere
- Added CD pipeline for aarch64
- Added pypi doc file for aach64 wheel

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: tools/pip/doc/AARCH64_CPU_ADDITIONAL.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+redistribute `libquadmath.so.0` as part of the Pypi package and users must
+manually install it.
+
+To install for other platforms (e.g. Windows, Raspberry Pi/ARM) or other versions, check [Installing MXNet](https://mxnet.apache.org/versions/master) for instructions on building from source.
+
+Installation
+------------
+To install, use:
+```bash
+pip install mxnet-aarch64
```
Comment (marcoabreu) [inline tools/pip/doc/AARCH64_CPU_ADDITIONAL.md:44]: Why create a separate package instead of adding aarch64 as a variant to the mxnet package?

Comment (mseth10) [reply tools/pip/doc/AARCH64_CPU_ADDITIONAL.md:44]: thanks for pointing it out. I did that by mistake. `pip install mxnet` should work.

**Review by Zha0q1 — APPROVED** [review]
LGTM!

**Review by szha — DISMISSED** [review]
Thanks Manu for adding the arm binary distribution support! One concern I have is the choice to increase the number of pip packages for mxnet.

File: tools/pip/setup.py [hunk 2]
```diff
@@ -127,7 +127,7 @@
 package_name = 'mxnet'
 
 variant = os.environ['mxnet_variant'].upper()
-if variant != 'CPU':
+if not variant.endswith('CPU'):
     package_name = 'mxnet_{0}'.format(variant.lower())
```
[the remark this answers is not in the document]
Comment (mseth10) [reply tools/pip/setup.py:131]: Currently, the wheel that this pipeline would generate looks like `mxnet-1.9.0b20210520-py3-none-manylinux2014_aarch64.whl` where `py3-none-manylinux2014_aarch64` is one of the platform tag output of `python3 -m pip debug --verbose`. Does that look okay to you? @szha

[the remark this answers is not in the document]
Comment (szha) [reply tools/pip/setup.py:131]: Yes, that looks perfect. We can leave the package name as is

**Comment by szha** [comment]
Merged. Thanks @mseth10! Let's monitor the CD for a few days and make sure builds are smooth.

**Comment by Zha0q1** [comment]
thanks Manu!

**Pull request merged**